wiremock/mappings/app-proxy.json

```diff
@@ -0,0 +1,14 @@
+{

```

Comment:
To make sure I understand how this works, is this taking any request to webservices.securetrading.net (which is wiremock) and proxying it on to library.securetrading.net (which is the npm files hosted on an Nginx server). That way it looks like things like st.js, card-number.html etc come from the same domain as the ST server (which is how it actually will work)

If this is how it works this is a really nice neat solution 👍

Reply:
Yes, that's how this works :) The nginx servers is hosting the pre-built application (from dist folder)